app/src/main/java/com/example/scoutconcordia/CalendarActivity.java

```diff
@@ -42,9 +103,388 @@ public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
+
+            int startRow = eventStartRow(startDate, endDate);
+
+            String eventName = event.getSummary();
+            String eventLocation = event.getLocation();
+            String start = sdfTime.format(startDate);
+            String end = sdfTime.format(endDate);
+
+            if (date.equals(dateInfoObj[0][1])) {   // today
+                fillEvent(0, startRow, totalRows, eventName, eventLocation, start, end, getResources().getColor(R.color.shadeOne));
+            }
+            else if (date.equals(dateInfoObj[1][1])) { //tomorrow
+                fillEvent(1, startRow, totalRows, eventName, eventLocation, start, end, getResources().getColor(R.color.shadeTwo));
+            }
+            else if (date.equals(dateInfoObj[2][1])) { // day after
+                fillEvent(2, startRow, totalRows, eventName, eventLocation, start, end, getResources().getColor(R.color.shadeOne));
+            }
+            else if (date.equals(dateInfoObj[3][1])) {
+                fillEvent(3, startRow, totalRows, eventName, eventLocation, start, end, getResources().getColor(R.color.shadeTwo));
+            } else if (date.equals(dateInfoObj[4][1])) {
+                fillEvent(4, startRow, totalRows, eventName, eventLocation, start, end, getResources().getColor(R.color.shadeOne));
+            } else if (date.equals(dateInfoObj[5][1])) {
+                fillEvent(5, startRow, totalRows, eventName, eventLocation, start, end, getResources().getColor(R.color.shadeTwo));
+            }
+            else if (date.equals(dateInfoObj[6][1])) {
+                fillEvent(6, startRow, totalRows, eventName, eventLocation, start, end, getResources().getColor(R.color.shadeOne));
+            }
+        }
+    }
+
+    // sets the values for the date's row (first row) and day's row (second row) for all 7 columns
+    // first column are set to today's date and day respectively, next column is next day
+    // uses the dateInfoObj created with String[][] dateInfoObj() method to get date and day value for each of the days
+    private void fillDayDateRows(String[][] dateInfoObj){
+        Resources r = getResources();
+        String name = getPackageName();
+        for(int i = 0; i < 7; ++i){
+            int dayColumnId = r.getIdentifier("day" + i, "id", name);
+            int dateColumnId = r.getIdentifier("date" + i, "id", name);
+            TextView dayColumn = findViewById(dayColumnId);
+            TextView dateColumn = findViewById(dateColumnId);
+            dateColumn.setText(dateInfoObj[i][0]);
+            dayColumn.setText(dateInfoObj[i][2]);
+        }
+    }
+
+    // returns a multidimensional array containing string values for the day and date from current day up to the 7th day
+    //String[][] dateInfoObj= new String[7][3]; // [x][0] = date (dd), [x][1] = date (yyyy-MM-dd), [x][2] = name of day, where x = 0 is today, x =1 is tomorrow and so on
```

Comment:
Coding convention:
can you split this comment so it fits the screen and we don't have to scroll horizontally?

Reply:
Sure.